### PR TITLE
Show Buildings That Never Submitted

### LINKED ISSUE #170: Show Buildings In The Benchmarking Data That Never Submitted
In issue #167, Curt brought up that his building (811 West Agatite Ave, building ID 256534) wasn't showing on our site and met the benchmarking requirements. It turns out it was in the raw data, but had _never_ reported. We should figure out a way to have those buildings included in our data, so that we can note these scofflaw buildings who have a 0% compliance rate.

**Note:** In the 2023 data this building starts reporting, so we may need to find other buildings for testing if we do the data update first.

### PR DESCRIPTION
**Pull request by JonHunt1995**
# Description

Updated data pipeline to no longer filter out buildings that have never reported data to the city. Added `❌` emoji to denote buildings that have never reported data, and added an infographic for building details as well. Finally, added logic to "Out of Date Data" to be mutually exclusive with "Building Never Submitted Data" status.

**Note:** Never submitted buildings show up in search and on the wards pages, which then lets alders see those buildings, and increases the count. E.g. for 40:

| Before | After |
| --- | --- |
| image | image |

**Also fixes reporting grade** - some not submitted buildings were getting grades due to being counted as reporting in years with no emissions, so Viktor fixed that as well, closing #288 

| Before | After |
| --- | --- |
| image | image |

Fixes #170

# Testing Instructions

I looked at a few buildings that had been marked as "❌" to check to see if they don't currently show up with the live website and that the years reported report card matches up with the "Building Never Submitted Data" status.

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] If I added a large new feature, I added it to the release notes (ReleaseNotes.vue)

## Data Update (if applicable):

- [ ] I have followed the [Data Update Checklist](DATA_UPDATE_CHECKLIST.md) for updating to new year's data

### CODE REVIEW

**Comment by JonHunt1995** [comment]
I have just noticed that a lot of the time the "never reported" buildings also commonly flag as "electric only" and I should probably change that so that doesn't happen.

**Comment by vkoves** [comment]
@JonHunt1995 - this PR is in draft, but did you mean for me to review it? If so, just hit "Ready for review" at the bottom next time or tag me!

**Comment by vkoves** [comment]
Known issues:

- [ ] Search excludes buildings that haven't submitted, but all emissions page shows them
- [ ] Homepage missing buildings that submitted in the past but not recently
- [ ] Need basic UI for never submitted building (e.g. drop all panels and show basic text)

**Comment by vkoves** [comment]
@JonHunt1995 - wanted to check in on this, any updates?

**Comment by vkoves** [comment]
Added a never submitted buildings route `/never-submitted` and added it to about, and polished up those building!

image


**Example:**

image

File: tests/data/scripts/unit/test_grade_buildings.py [hunk 1]
```diff
@@ -198,6 +222,12 @@ def test_energy_mix_grading():
     assert result.loc[result["ID"] == 3, "EnergyMixWeightedPctSum"].iloc[0] == 0.0
     assert result.loc[result["ID"] == 4, "EnergyMixWeightedPctSum"].iloc[0] == 50.0
 
+    # Building 5: no energy data at all shouldn't be graded as if it had a real 0% mix
+    id5_row = result[result["ID"] == 5]
+    assert pd.isna(id5_row["EnergyMixWeightedPctSum"].iloc[0])
+    assert pd.isna(id5_row["EnergyMixPercentileGrade"].iloc[0])
+    assert pd.isna(id5_row["EnergyMixLetterGrade"].iloc[0])
+
```
Comment (RamarkoB) [inline tests/data/scripts/unit/test_grade_buildings.py:261]: Add a one line to check that this find that there is no first year recorded

File: tests/data/scripts/unit/test_grade_buildings.py [hunk 2]
```diff
@@ -198,6 +222,12 @@ def test_energy_mix_grading():
...
```
Comment (RamarkoB) [inline tests/data/scripts/unit/test_grade_buildings.py:262]: Add test to check that `SubmittedRecordsLetterGrade` is `none`, as the corresponds to never submitted

**Comment by RamarkoB** [comment]
Screenshot 2026-08-04 at 8 16 10 PM
Screenshot 2026-08-04 at 8 17 51 PM
